### DISCUSSION TITLE
find_diff_features

### DISCUSSION THREAD

**Discussion by TingTingShao** [aertslab/pycisTopic#136] [Q&A] [answered]
Hi, 

I have questions regarding find_diff_features. I am not sure how to interpret the contrasts. 

I am not sure if I understand it correctly, when the `constrasts=None`, the comparison will be 
1 vs 2, 3
2 vs 1, 3
3 vs 1, 2

If I have three clusters. 

Thanks,
tingting

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi TingTing

I converted your issue in a discussion.

> I am not sure if I understand it correctly, when the constrasts=None, the comparison will be
1 vs 2, 3
2 vs 1, 3
3 vs 1, 2
If I have three clusters.

Indeed, that's correct! 

Best,

Seppe
